test/OAuth2/Storage/MongoDBTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace OAuth2\Storage;
+
+/**
+ * Description of MongoDBTest
+ *
+ * @author Roman Shuplov  <astronin@gmail.com>
+ */
+class MongoDBTest extends BaseTest 
+{
+    
+    public function __construct() {
+        $mongodb = Bootstrap::getInstance()->getMongoDB();
+    }
+    
+    public function testSetClientDetails() {
```

Comment:
curly braces should be on their own line, to be consistent with the styles in the rest of this library